src/screens/PhotosPermissionsScreen/index.tsx

```diff
@@ -24,9 +45,14 @@ function PhotosPermissionsScreen({ navigation }: PhotosScreenProps<'PhotosPermis
   ];
 
   useEffect(() => {
-    photosCtx.permissions.getPermissionsStatus().then((permissions) => {
-      setPermissions(permissions);
-    });
+    photosCtx.permissions
+      .getPermissionsStatus()
+      .then((permissions) => {
+        setPermissions(permissions);
```

Comment:
This is a good example of applying SRP, `getPermissions` gives the permissions and another function is responsible for handling that information in any use-case desired way.